src/logic.rs

```diff
@@ -142,8 +142,14 @@ pub fn get_deck_assoc_filename(deck_id: &Deck, decks: &HashMap<Deck, DeckStatus>
         trace!("Get associated filename of deck {}: {}", deck_id, fpath);
         let file_path = Path::new(&fpath);
         let subtitle_filename = file_path.file_stem();
-        debug!("Returning filename {}", subtitle_filename.unwrap().to_os_string().into_string().unwrap());
-        return Some(subtitle_filename.unwrap().to_os_string().into_string().unwrap());
+        if subtitle_filename.is_some() {
+            debug!("Returning filename {}", subtitle_filename.unwrap().to_os_string().into_string().unwrap());
+            return Some(subtitle_filename.unwrap().to_os_string().into_string().unwrap());
+        }
+        else {
+            debug!("Filename not found, returning None");
+            None
+        }
```

Comment:
Is there a case where `file_stem` returns None here?

Also 
```suggestion
   match subtitle_filename {
        Some(subtitle_filename) => {
            debug!("Returning filename {}", subtitle_filename.unwrap().to_os_string().into_string().unwrap());
            return Some(subtitle_filename.unwrap().to_os_string().into_string().unwrap());
        },
        None => {
            debug!("Filename not found, returning None");
            None
        }
    }
```

Reply:
> Is there a case where `file_stem` returns None here?
> 
> Also

Same answer for `trackPaused`, requires the logic filter change, if the song is removed from the deck and the crossfader has this deck active in the mix, you may get an empty filepath.

Reply:
For this one it's easier since it's  a new API. However please check that other APIs don't break in a similar fashion. (I assume not since they are supposed to deal with file-not-founds to begin with)